ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -705,28 +633,28 @@ + (void)updateViewController:(UIViewController *)vc
 
 + (void)addSubviewsToNavItem:(UINavigationItem *)navitem withConfig:(RNSScreenStackHeaderConfig *)config
```

Comment:
Why is this method needed?

Reply:
[4e36439](https://github.com/software-mansion/react-native-screens/pull/1418/commits/4e364397a17b95285d0eacf02aadfd8133b584e5)